schemas/1.0/maat/means.json

```diff
@@ -334,6 +361,14 @@
                   "month",
                   "annual"
                 ]
+              },
+              "ownership_type":{
+                "type":"string",
+                "enum":[
+                  "applicant",
+                  "partner",
+                  "applicant_and_partner"
```

Comment:
Can an employment be applicant_and_partner? Job share? I'd imagine they'd need to be listed separately. Perhaps best to omit "applicant_and_partner".

Reply:
Wider team question here as to what we want to do when a user changes their answer to a fundamental question like whether their client has a partner.. do we want to reset answers? do we want to handle changes ourselves e.g. update ownership to applicant and partner. This has wider implications outside of just the payments attributes

Reply:
Decided to remove `ownership_type` from outgoings payments as it will only ever be `applicant_and_partner`

Reply:
Assuming you left it on outgoings_payments and this comment is outdated. They can be applicant, or applicant_and_partner

Reply:
Yes that's right i keep getting confused 😫